src/Components/ApiFetch/RoversApiFetch.tsx

```diff
@@ -0,0 +1,15 @@
+﻿import React from "react";
+
+export interface RoverImages {
+    img_src: string;
+    earth_date: string;
+    sol: number;
+}
+
+
+export async function getRoverImages(dateInput: string | null): Promise<RoverImages> {
```

Comment:
Same as for the interface name, if it's only actually returning one image.

Reply:
Agreed. currently only returning one so have changed.